### DISCUSSION TITLE
How to redirect renamed pages

### DISCUSSION THREAD

**Discussion by peterpeterparker** [facebook/docusaurus#8825] [General]
In my documentation I got an existing doc page accessible through the URL `https://juno.build/docs/category/add-juno-to-an-app`. I would like to rename it to `.../add-juno-to-YOUR-app` but it means it would break back links and SEO.

Is there a way to deploy the new route and to redirect the old route to the new one without server side redirect configuration?

**Comment by slorber** [comment]
> without server side redirect configuration

Not doing so with server redirects is not the bet SEO option, so if you care about SEO you need server redirects.

If you really want to do a client redirect, you can do so with some JS in a file `static/docs/category/add-juno-to-an-app.html`. That's what our client redirect plugin does: https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-client-redirects

**Reply by peterpeterparker** [reply]
Gotcha, thanks
